test/gpc.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert')
```

Comment:
Please could you add a test for the relative rule priorities of `GPC_HEADER_PRIORITY` in `test/rulePrioirities.js` too?